.ci/ci.yaml

```diff
@@ -0,0 +1,288 @@
+jobs:
+- job: Linux
+  pool:
+    vmImage: ubuntu-16.04
+  strategy:
+    matrix:
+      Plain:
+        CMakeArgs: '-DDEBUG=ON -DPROFILE=OFF -DBUILD_PYTHON_BINDINGS=OFF'
+        python.version: '2.7'
+      Python27:
+        python.version: '2.7'
+        CMakeArgs: '-DDEBUG=OFF -DPROFILE=OFF -DPYTHON_EXECUTABLE=/usr/bin/python'
+      Python37:
+        python.version: '3.7'
+        CMakeArgs: '-DDEBUG=OFF -DPROFILE=OFF -DPYTHON_EXECUTABLE=/usr/bin/python3'
```

Comment:
It may also be worth adding a version for `BUILD_MARKDOWN_BINDINGS=ON` too, to make sure there are no problems there.

Reply:
Good idea.